src/components/Staking/StakingHowSoloWorks.tsx

```diff
@@ -50,7 +43,7 @@ const StakingHowSoloWorks: React.FC<IProps> = () => {
       justifyContent="space-between"
     >
       <OrderedList listData={items} />
-      <GatsbyImage image={getImage(image)!} alt="" />
+      <Image src="/hackathon_transparent.png" alt="" width={400} height={400 / 1.5}/>
```

Comment:
We should do a static import here and only set the `width`. The height is automatically calc by Next.

```tsx
import hackathon from "@/public/hackathon_transparent.png"

<Image src={hackathon} alt="" width={400} />
```